src/fetch/ipfs.py

```diff
@@ -129,18 +129,18 @@ async def fetch_many(  # pylint: disable=too-many-locals
                     max_retries, previous_attempts, session, first_seen_block
                 )
```

Comment:
I thought the plan was to first look in the backend?

Reply:
I just switched around trying to fetch backend with trying to fetch old schema, then realized it didn't even help.